### DISCUSSION TITLE
Add additional "copy URL" shortcuts

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#794] [Ideas]
### What feature would you like to see?

One of the features I use most in Arc is the 'copy URL as markdown' which allows me to copy the page title and url into markdown formatting

On mac, that default shortcut is `cmd+shift+option+v`

When I use it on Zen Browser's homepage, here is what it pastes: `[Zen Browser](https://www.zen-browser.app/)`

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/576_

**Comment by jemjam** [comment]
Let me add a few small details about how Arc does this (since it's one of my favorite and most used features).

`cmd-shift-c` is the default to copy the current tab (url only) to clipboard.
`cmd-shift-option-c` will copy it as a markdown link:  `[Tab Title](url)`

Both of these keys work with one (or more) tabs selected. When multiple tabs are selected, each entry is on a new line.

As a browser user, I like to be able to copy links quickly to share in other apps or my notes. Being able to open a bunch of tabs then save the whole list quickly is such a killer browser feature. I'm surprised we haven't all been doing this sooner.

**Comment by drfiresign** [comment]
Being able to do this with multiple tabs would be phenomenal! I didn't know you could copy as markdown with Arc either, and that would actually make writing documentation for my work significantly easier.

**Comment by theherk** [comment]
There is a plugin, [asamuzaK/url2clipboard: Copy document URL / link URL to clipboard as HTML, Markdown, Plain Text, etc.](https://github.com/asamuzaK/url2clipboard) / [Copy URL To Clipboard – Get this Extension for 🦊 Firefox (en-US)](https://addons.mozilla.org/en-US/firefox/addon/url2clipboard/)

With this you can set it to generate links based on shortcuts in many different formats: wiki, markdown, plain text, etc. For me, `⌘ ⇧ m` get the markdown link with text (as used above) and `⌘ ⇧ u` gets just the url.

It of course would be nice to have built in, but this works well.
